dfttopif/parsers/vasp.py

```diff
@@ -14,34 +14,55 @@ class VaspParser(DFTParser):
     Parser for VASP calculations
     '''
 
+    def __init__(self, directory):
+        super(VaspParser, self).__init__(directory)
+
+        # Get the list of files in this directory
+        files = [f for f in os.listdir(directory) if os.path.isfile(os.path.join(directory, f))]
+
+        # Find the outcar file
+        def _find_file(name):
+            """Find a filename that contains a certain string"""
+            for f in files:
+                if name in f.upper():
```

Comment:
this would also match `"this_is_not_an_outcar"`.  If the PR is to support extensions, could you use `f.upper().startswith("OUTCAR")` or equivalent?

Reply:
Good point :). I'll make a fix.